src/workerd/api/gpu/gpu-bindgroup-layout.c++

```diff
@@ -25,9 +25,15 @@ wgpu::BufferBindingType parseBufferBindingType(kj::StringPtr bType) {
 wgpu::BufferBindingLayout parseBufferBindingLayout(GPUBufferBindingLayout& buffer) {
   wgpu::BufferBindingLayout l;
```

Comment:
non-blocking: I know this isn't new here but could this be named something more descriptive than just `l` ? :-)